### PR TITLE
fixes #463 move details drawer to the left 

### LINKED ISSUE #463: move the probe details column to the left of the content column
Stretch goal...have it tuck away below the content on smaller viewports.

### PR DESCRIPTION
**Pull request by spasovski**
image

### CODE REVIEW

File: public/static/global.css [hunk 1]
```diff
@@ -891,10 +893,6 @@ tr .data-cell--header:first-child {
   text-align: center;
 }
 
-/* default */
-.data-cell--align-undefined {
-}
```
Comment (openjck) [inline public/static/global.css:896]: 🎉

File: src/routing/wrappers/Layout.svelte [hunk 2]
```diff
@@ -29,9 +32,9 @@
       <MainSelectors />
     </ContentHeader>
     <ContentBody>
-      <div class="graphic-body">
+      <div class="graphic-body" class:graphic-body-home={isHomePageView}>
         <slot />
-        {#if $store.route.section === 'probe'}
+        {#if !isHomePageView}
```
Comment (openjck) [inline src/routing/wrappers/Layout.svelte]: What's the benefit of reversing this logic? We're rendering something that only makes sense in the probe section.

File: src/routing/wrappers/Layout.svelte [hunk 3]
```diff
@@ -14,6 +14,9 @@
   import ProbeDetails from '../../components/regions/ProbeDetails.svelte';
 
   import { store } from '../../state/store';
+
+  let isHomePageView = true;
```
Comment (openjck) [inline src/routing/wrappers/Layout.svelte]: It looks like we don't need this `let`.

**Review by openjck — CHANGES_REQUESTED** [review]
r+wc

File: src/routing/wrappers/Layout.svelte (hunk 3 above)
Comment (spasovski) [reply src/routing/wrappers/Layout.svelte]: Yea I suppose this should be `isProbeDetailsView` instead so we don't have to check the inverse.

**Pull request merged**